docs/ref-datastore-manipulating-data.md

```diff
@@ -9,34 +9,28 @@ All interaction is done with the on-device storage.
 ## Saving data
 
 ```typescript
-import { save } from 'offix-datastore';
+import { TaskModel } from 'datastoreConfig';
 
-save({
+TaskModel.save({
     title: "Write Docs",
     description: "Write Offix Docs",
-    __typename: "Task"
 }).then((data) => {
-    console.log(data); // { id: '...', title, description, __typename  }
+    console.log(data); // { id: '...', title, description }
 })
 ```
 
 ## Querying data
 
 ```typescript
-import { query } from 'offix-datastore;
+import { TaskModel } from 'datastoreConfig';
 
-query({
-    __typename: "Task"
-}).then((data) => {}) // Retrieves all tasks
+TaskModel.query().then((data) => {}) // Retrieves all tasks
 ```
 
 Predicate functions are used to filter data
 
 ```typescript
-query({ 
-    title: "",
-    __typename: "Task"
-}, (p: any) => p.title("eq", "test"))
+TaskModel.query((p: any) => p.title("eq", "test"))
```

Comment:
Again, much easier to read, but also I think a better dev experience. Having to specify the `__typename` like that in every query muddies the code up. This is a much better approach because it abstracts away the need to specify the model type like that.